critbit-test-tree.h

```diff
@@ -30,8 +30,6 @@
 #ifndef	_SYS_TREE_H_
 #define	_SYS_TREE_H_
 
-#include <sys/cdefs.h>
```

Comment:
Could you please put it __FreeBSD__ around it instead.

Reply:
done